lib/workers/repository/onboarding/branch/index.spec.ts

```diff
@@ -245,5 +258,75 @@ describe('workers/repository/onboarding/branch/index', () => {
       expect(git.checkoutBranch).toHaveBeenCalledTimes(1);
       expect(git.commitFiles).toHaveBeenCalledTimes(0);
     });
+
+    describe('tests onboarding rebase/retry checkbox handling', () => {
+      beforeEach(() => {
+        GlobalConfig.set({ platform: 'github' });
+        config.onboardingRebaseCheckbox = true;
+        OnboardingState.prUpdateRequested = false;
+        git.getFileList.mockResolvedValueOnce(['package.json']);
+        platform.findPr.mockResolvedValueOnce(null);
+        rebase.rebaseOnboardingBranch.mockResolvedValueOnce(null);
+      });
+
+      afterAll(() => {
+        delete config.onboardingRebaseCheckbox;
+      });
```

Comment:
why is this needed?

Reply:
```suggestion
      afterAll(() => {
        config.onboardingRebaseCheckbox = false;
      });
```
maybe?

i wanted to disable it for future tests where this might interfere (being `true`)

Reply:
checkout the beforeEach, config is always fresh for each test, so this does nothing